pages/index.js

```diff
@@ -161,9 +164,19 @@ export default class Index extends React.PureComponent {
         scrollY: 0,
     };
 
+    propTypes = {
+        animationBackground1: PropTypes.array.isRequired,
+    };
+
     mountainSlideScrollDividend = 50;
     citySlideScrollDividend = 30;
 
+    static async getInitialProps() {
+        const response = await fetch('http://127.0.0.1:8001/.json');
```

Comment:
NODE_ENV=staging next start 
and then add a config.js with a if (NODE_ENV=== staging) else ....